app/controllers/emails_controller.rb

```diff
@@ -8,7 +8,7 @@ def show
     application_letter_status = get_corresponding_application_letter_status
     @addresses = @event.email_addresses_of_type(application_letter_status)
 
-    @email = Email.new(hide_recipients: true, reply_to: 'workshop.portal@hpi.de', recipients: @addresses.join(','),
+    @email = Email.new(hide_recipients: true, reply_to: Rails.configuration.from_address, recipients: @addresses.join(','),
```

Comment:
I think it was already mentioned in another PR(#355) that this value should be in the config, not hardcoded. However we use workshop.portal@gmail.com as address to send mails atm, and the default reply to is workshop.portal@hpi.de.
TLDR: I agree its good to read this value from the config, but this is the wrong config value